imi/src/main/java/org/motechproject/nms/imi/service/impl/ChecksumHelper.java

```diff
@@ -0,0 +1,17 @@
+package org.motechproject.nms.imi.service.impl;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+
+public final class ChecksumHelper {
+
+    private ChecksumHelper() { }
+
+    public static String checksum(File file) throws IOException {
+            return DigestUtils.sha1Hex(new FileInputStream(file));
```

Comment:
Nothing wrong here... but if this is just a 1 line return, can we just replace the call line to this function with the return line here? :thought_balloon: